fastlane/Fastfile

```diff
@@ -583,13 +547,22 @@ platform :android do
     end
   end
 
-  #####################################################################################
-  # Screenshots lanes
-  #####################################################################################
+  desc 'Trigger a release build on Buildkite'
+  lane :trigger_release_build do |branch_to_build: git_branch|
+    UI.important("There will be no release build from #{branch_to_build}.")
```

Comment:
Suggesting this tweak because I found it confusing to say "there will be no release build", as it might suggest that we have decided not to make a release build at all and the Release Manager should thus not expect a release build to be done for this sprint… at all (as opposed to not expect it to be automated, and will have to still do a release build… but manually)
```suggestion
    UI.important("There will be no release build triggered automatically from #{branch_to_build}.")
```